documentation/shared/attributes.adoc

```diff
@@ -164,6 +163,7 @@
 :BookURLDeploying: ./deploying.html
 :BookURLUsing: ./using.html
 :BookURLOverview: ./overview.html
+:book-bridge: ./bridge.html
```

Comment:
Assuming we take the same approach, but _`bridge.html` doesn't render correctly at the moment.